website/docs/reference/notifications.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Notifications
+---
+
+:::info Availability
+
+Notifications were introduced in Unleash 4.22.0 for pro and enteprise customers.
+
+:::
+
+Unleash's notifications give you updates when certain events occur in projects that you are part of. The notifications are accessible from the notifications button in the navigation section of the admin UI.
+
+Your notifications will only contain updates from other members; you will not get notifications about actions you perform yourself.
+
+![The notifications overview. It's telling the user about a feature that was enabled in production and that there is a change request ready for review.](/img/notifications.png)
+
+From within the notifications list, you can:
+* navigate to the features or change request that the notification is for
+* mark all notifications as read
+* filter out unread notifications
+
```

Comment:
Can you disable notifications (either for a project or as a whole) today? Do we plan to add that later?

```suggestion
Notifications can not be disabled.

```

Reply:
Currently there's no way to disable notifications so your comment is spot on. We're considering adding configuration options based on the adoption of the capability.